.github/actions/slack-message/action.yml

```
@@ -0,0 +1,38 @@
+name: Send a message to slack
+
+inputs:
+  commit_url:
+    description: normally "${{ github.server_url }}/${{ github.repository }}/commit/${{ GITHUB_SHA}}"
+  job_status:
+    description: passthrough for job.status
+  msg:
+    description: The slack message text
+  run_url:
+    description: normally "${{ GITHUB_REF#refs/heads/ }}-${{ GITHUB_SHA }}"
+  sha:
+    description: normally "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+  slack_url:
+    description: passthrough for secrets.SLACK_WEBHOOK_URL
+
+runs:
+  using: composite
+  steps:
+    - id: slack-message
+      uses: slackapi/slack-github-action@v1.24.0
+      with:
+        payload: |
+          {
+            "text": "${{ inputs.text }} :: build: ${{ inputs.job_status }} - SHA: ${{  inputs.sha_info.outputs.SHA }}",
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "${{ inputs.text }} :: <${{  inputs.sha_info.outputs.RUN_URL }}|[Logs]> <${{ inputs.sha_info.outputs.COMMIT_URL }}|[Base]>\nCommit: <${{  inputs.sha_info.outputs.COMMIT_URL }}|${{  inputs.sha_info.outputs.SHA }}>"
```

I think this should be `inputs.msg`?
```suggestion
                  "text": "${{ inputs.msg }} :: <${{  inputs.sha_info.outputs.RUN_URL }}|[Logs]> <${{ inputs.sha_info.outputs.COMMIT_URL }}|[Base]>\nCommit: <${{  inputs.sha_info.outputs.COMMIT_URL }}|${{  inputs.sha_info.outputs.SHA }}>"
```

---

whoops, thanks!